opsdroid/parsers/watson.py

```diff
@@ -0,0 +1,95 @@
+"""A helper function for parsing and executing IBM watson skills."""
+import logging
+import json
+
+from ibm_watson import AssistantV2
+from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
+from ibm_watson import ApiException
+
+from opsdroid.const import WATSON_API_ENDPOINT
+from opsdroid.const import WATSON_API_VERSION
```

Comment:
You have this import twice, you could combine them into 

```suggestion
from opsdroid.const import WATSON_API_ENDPOINT, WATSON_API_VERSION
```